6_object_wrap/nan/package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "gypfile": true,
   "dependencies": {
-    "bindings": "~1.2.1",
-    "nan": "^2.0.0"
+    "bindings": "~1.5.0",
```

Comment:
My thought is that if we need to run this code on CI it could be good run it against the last version of the modules in the dependencies. What do you think to set the version to `*`. Is there some problem that I'm not considering?

Reply:
When semver version release, the old code may not compile.